City-of-Helsinki/berth-reservations-admin
# File upload refactoring

## Description :sparkles:

* Refactored FileUpload into separate FileList and FileUpload components
* Use HDS Button in FileUpload
* Adjusted icons stories typing

## Issues :bug:

### Closes :no_good_woman:

* [VEN-575](https://helsinkisolutionoffice.atlassian.net/browse/VEN-575): Single harbor page: edit harbor properties

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️

* Tests have been updated, and should have 100% coverage

### Manual testing :construction_worker_man:

* Go to an individual harbor's page and edit the file properties
  * File upload should create file entities correctly, but the actual data is not correctly saved into backend (needs further investigation)
* The components can be tested in Storybook as well (FileList, FileUpload)

## Screenshots :camera_flash:

## Additional notes :spiral_notepad:
